src/legendmeta/textdb.py

```diff
@@ -401,12 +401,12 @@ def on(
         system: 'all', 'phy', 'cal', 'lar', ...
             query only a data taking "system".
         """
-        jsonl = self.__path__ / "validity.jsonl"
-        if not jsonl.is_file():
-            msg = f"no validity.jsonl file found in {self.__path__!s}"
+        yml = self.__path__ / "validity.yml"
```

Comment:
you might want to use this dictionary: 

https://github.com/legend-exp/pylegendmeta/blob/03da06e86a98b86844cfa1d6ee182ec7a4be2da9/src/legendmeta/utils.py#L26